### PR TITLE
feat: [CHATS-353] add location for chats events websocket

### PR DESCRIPTION
**Pull request by lucagasparini1981**
I've added locations for /services/chats/events to proxy pass a websocket

### CODE REVIEW

**Review by frisonisland — APPROVED** [review]
LGTM!

**Review by frisonisland — CHANGES_REQUESTED** [review]
My bad, could you merge it into devel branch?

**Review by frisonisland — APPROVED** [review]

File: conf/nginx/templates/nginx.conf.web.http.default.template [hunk 1]
```diff
@@ -50,6 +50,17 @@ server
         proxy_http_version 1.1;
     }
 
+    location /services/chats/events
```
Comment (keshavbhatt) [inline conf/nginx/templates/nginx.conf.web.http.default.template:53]: Is the absence of a following slash intended and verified to ensure that we do not receive an unnecessary redirect in response to the request recieved at this endpoint?

ref: [from nginx docs](http://nginx.org/en/docs/http/ngx_http_core_module.html#location), 
>If a location is defined by a prefix string that ends with the slash character, and requests are processed by one of [proxy_pass](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass), [fastcgi_pass](http://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_pass), [uwsgi_pass](http://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_pass), [scgi_pass](http://nginx.org/en/docs/http/ngx_http_scgi_module.html#scgi_pass), [memcached_pass](http://nginx.org/en/docs/http/ngx_http_memcached_module.html#memcached_pass), or [grpc_pass](http://nginx.org/en/docs/http/ngx_http_grpc_module.html#grpc_pass), then the special processing is performed. In response to a request with URI equal to this string, but without the trailing slash, a permanent redirect with the code 301 will be returned to the requested URI with the slash appended

**Review by keshavbhatt — COMMENTED** [review]
Requested a little check to confim the behaviour of this addition. Everything else looks fine to me.
Please make sure to check before merging.

File: conf/nginx/templates/nginx.conf.web.http.default.template (hunk 1 above)
Comment (lucagasparini1981) [reply conf/nginx/templates/nginx.conf.web.http.default.template:53]: This endpoint is related to a websocket and we don't want anything after "events".

**Review by keshavbhatt — APPROVED** [review]
lgtm

**Pull request merged**